redaxo/src/addons/mediapool/lib/media.php

```diff
@@ -325,7 +325,7 @@ public static function isImageType($extension)
 
     public function hasValue($value)
     {
-        return isset($this->$value);
+        return isset($this->$value) || isset($this->{'med_' . $value});
```

Comment:
Musst nun getValue() anpassen, da er jetzt nie mehr in zeile 338 reinkommt

Reply:
Am besten in getValue() direkt mit isset() arbeiten statt hasValue()

Reply:
stimmt